pkg/liqoctl/status/peer/peerinfo.go

```diff
@@ -0,0 +1,277 @@
+				continue
+			} else {
+				found = true
+			}
+			foreignCluster, err := getters.GetForeignClusterByClusterID(ctx, pic.options.CRClient, te.Spec.ClusterIdentity.ClusterID)
+			if err != nil {
+				pic.addCollectionError("ForeignCluster", "unable to collect ForeignCluster", err)
+			}
+
+			clusterSection := pic.peerInfoSection.AddSection(te.Spec.ClusterIdentity.ClusterName)
+
+			pic.addPeerSection(clusterSection, foreignCluster)
+
+			pic.addAuthSection(clusterSection, foreignCluster)
+
+			err = pic.addNetworkSection(ctx, clusterSection, foreignCluster, te, localClusterName)
+			if err != nil {
+				pic.addCollectionError("Network", "unable to collect Network", err)
+			}
+
+			err = pic.addResourceSection(ctx, clusterSection, remoteClusterID, localClusterName, remoteClusterName)
+			if err != nil {
+				pic.addCollectionError("Resources", "unable to collect Resources", err)
+			}
+		}
+	}
+
+	if !found {
+		pic.addCollectionError("", "", fmt.Errorf("unable to find peers: \"%s\"", strings.Join(pic.options.RemoteClusterNames, `","`)))
+	}
+
+	return nil
+}
+
+// addPeerSection adds a section about the peering generic info.
+func (pic *PeerInfoChecker) addPeerSection(rootSection output.Section, foreignCluster *discoveryv1alpha1.ForeignCluster) {
+	peerSection := rootSection.AddSection("Peering")
+	peerSection.AddEntry("Type", string(foreignCluster.Spec.PeeringType))
+
+	directionSection := peerSection.AddSection("Direction")
+	incomingStatus := peeringconditionsutils.GetStatus(foreignCluster, discoveryv1alpha1.IncomingPeeringCondition)
+	directionSection.AddEntry("Incoming", string(incomingStatus))
+	outgoingStatus := peeringconditionsutils.GetStatus(foreignCluster, discoveryv1alpha1.OutgoingPeeringCondition)
+	directionSection.AddEntry("Outgoing", string(outgoingStatus))
+}
+
+// addAuthSection adds a section about the authentication status.
+func (pic *PeerInfoChecker) addAuthSection(rootSection output.Section, foreignCluster *discoveryv1alpha1.ForeignCluster) {
+	authSection := rootSection.AddSection("Authentication")
+	authStatus := peeringconditionsutils.GetStatus(foreignCluster, discoveryv1alpha1.AuthenticationStatusCondition)
+	authSection.AddEntry("Status", string(authStatus))
+	if pic.options.Verbose {
+		authSection.AddEntry("Auth URL", foreignCluster.Spec.ForeignAuthURL)
+		if foreignCluster.Spec.ForeignProxyURL != "" {
+			authSection.AddEntry("Auth Proxy URL", foreignCluster.Spec.ForeignProxyURL)
+		}
+	}
+}
+
+// addNetworkSection adds a section about the network configuration.
+func (pic *PeerInfoChecker) addNetworkSection(ctx context.Context, rootSection output.Section,
+	foreignCluster *discoveryv1alpha1.ForeignCluster, tunnelEndpoint *netv1alpha1.TunnelEndpoint, localClusterName string) error {
+	networkSection := rootSection.AddSection("Network")
+	networkStatus := peeringconditionsutils.GetStatus(foreignCluster, discoveryv1alpha1.NetworkStatusCondition)
+	networkSection.AddEntry("Networking", string(networkStatus))
```

Comment:
```suggestion
	networkSection.AddEntry("Status", string(networkStatus))
```

For consistency with the authentication one.